.evergreen/config/templates/build/build-libmongoc.yml

```diff
@@ -4,7 +4,6 @@
 # These tasks are used in the build task group "build-php-libmongoc" defined in .evergreen/config/build-task-groups.yml,
 # where the "max_hosts" setting must be set to the number of enabled tasks.
 # Last but not least, make sure the corresponding test variants are enabled in .evergreen/config/test-variants.yml
-tasks:
```

Comment:
I decided to remove the top-level key in all templates to avoid having to use a yaml library to combine individual config files. It's not optimal, but it works for the time being.

Reply:
I think this works fine for the time being - the less I mess with the files the better it is.